common.mk

```diff
@@ -192,6 +192,7 @@ test-slow:  ##- Run slow tests
 
 .PHONY: test-coverage
 test-coverage:  ## Generate coverage report
+	uv run pytest -vv tests/unit
```

Comment:
You still have this, could it be why the coverage tests are failing in CI?

Reply:
This was added to identify the exact point coverage tests were failing, and can be removed now.